modules/benchmark/R/load.data.R

```diff
@@ -16,6 +16,7 @@ load.data <- function(data.path, format, start_year = NA, end_year=NA, site=NA,
   require(PEcAn.benchmark)
   require(lubridate)
   require(udunits2)
+  require(dplyr)
```

Comment:
Hi @bcow - no, the problem is that `require` isn't strict enough! That is, if the package isn't available, it simply returns FALSE and execution proceeds. But in all these cases, the code absolutely depends on that package; you want an error to occur if it isn't available.

Side note: this is **really** poor semantics by R. You'd think `require` would be the stricter one, right? It should be renamed `available` or something, and the confusingly-named `library` should have the name `require`.